### PR TITLE
Update pybabel files and localizations.

### PR DESCRIPTION
**Pull request by idosavion**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -229,7 +229,7 @@ <h3>אודות ANYWAY</h3>
...
למפה וכן תתאפשר יכולת לפתח דיונים מבוססי מיקום על פתרונות אפשריים והתארגנות אזרחית. באמצעות ידע זמין, נגיש
ושקוף אנו שואפים להעלות את המודעות לנושא תאונות הדרכים ולהעצים את יכולת ההשפעה של כל אחד ואחת על הסביבה
הקרובה- הבית, השכונה, בית הספר של הילדים, מקום העבודה ועוד- על ידי פעולה משותפת בקהילה. אנחנו מאמינים שבעזרת
ידע ציבורי קיים המוצג באתר יווצרו ידע ותובנות חדשים. שיתוף הידע הזה יאפשר לחברה האזרחית להתארגן לפעולה
אקטיבית ולהזרים הצעות, רעיונות והתראות בחזרה אל הרשויות.

+ANYWAY מציגה על גבי מפה תאונות דרכים עם נפגעים ע״ב נתוני הלשכה המרכזית לסטטיסטיקה, ונתונים המוזרמים בזמן אמת באדיבות ארגון ״איחוד הצלה״. בהמשך, יתווסף מידע נוסףלמפה וכן תתאפשר יכולת לפתח דיונים מבוססי מיקום על פתרונות אפשריים והתארגנות אזרחית. באמצעות ידע זמין, נגישושקוף אנו שואפים להעלות את המודעות לנושא תאונות הדרכים ולהעצים את יכולת ההשפעה של כל אחד ואחת על הסביבההקרובה- הבית, השכונה, בית הספר של הילדים, מקום העבודה ועוד- על ידי פעולה משותפת בקהילה. אנחנו מאמינים שבעזרתידע ציבורי קיים המוצג באתר יווצרו ידע ותובנות חדשים. שיתוף הידע הזה יאפשר לחברה האזרחית להתארגן לפעולהאקטיבית ולהזרים הצעות, רעיונות והתראות בחזרה אל הרשויות.
```
Comment (elazarg) [inline templates/index.html:232]: Whitespace?

File: templates/index.html [hunk 2]
```diff
@@ -463,9 +463,9 @@ <h5>הנך מוזמנ/ת להשאיר פרטים על מנת שנוכל לעדכ
         <div class="modal-header">
             <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
             {% if current_user.is_authenticated %}
-                <h3>{{ gettext('התנתק') }}</h3>
+                <h3>{{ _('log out') }}</h3>
```
Comment (elazarg) [inline templates/index.html:466]: There's already "Logout" and "Login" in the `messages` file

**Review by elazarg — APPROVED** [review]

**Comment by idosavion** [comment]
@atalyaalon - can we merge this one after rebase?

**Comment by atalyaalon** [comment]
@idosavion sure, do you want to resolve conflicts?

File: templates/index.html (hunk 1 above)
Comment (idosavion) [reply templates/index.html:232]: fixed some malformed characters

Comment (elazarg) [reply templates/index.html:232]: Looks like a missing space.

Comment (idosavion) [reply templates/index.html:232]: will be fixed in a future PR, much more work needed there

**Pull request merged**